azure-devops/azext_devops/dev/pipelines/pipeline_create.py

```diff
@@ -71,6 +72,9 @@ def pipeline_create(name, description=None, repository=None, branch=None, yml_pa
     :type service_connection: str
     :param queue_id: Id of the queue in the available agent pools. Will be auto detected if not specified.
     :type queue_id: str
+    :param skip_first_run: Specify this flag to prevent the first run being triggered by the command.
```

Comment:
will suggest default to be false, or have `queue_first_run` as a param.
The command is to create pipeline so doing anything above that (like queue it) should be optional (opt-in not opt-out)

Reply:
Resolving since I have captured all feedback under one comment.